service/provider/src/main/java/gov/va/vro/service/provider/services/MasAssessmentResultProcessor.java

```diff
@@ -24,5 +24,9 @@ public void process(Exchange exchange) throws Exception {
       return;
     }
     saveToDbService.insertAssessmentResult(evidence, diagnosticCode);
+    String claimSubmissionId = evidence.getClaimSubmissionId();
+    Boolean sufficiencyFlag = evidence.isSufficientForFastTracking();
+    saveToDbService.updateSufficientEvidenceFlag(
```

Comment:
It looks like that the two variables created here don't do much. Just using evidence get methods in next line can reduce a couple of lines.